recipes/gnumpy/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "gnumpy" %}
+{% set version = "0.2" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "070b78ee648c42e3dcd2f56a4308cc26bcaa1de30a4f7c078eb5828050a4ec4f" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - pip
+    - python <3
```

Comment:
Are you sure that you want to package it just for Python2.7 ?

Reply:
The software is so old, I presume it only works for 2.7

Reply:
Just to confirm, because Python 2.7 is reaching the end of his life in January. :smile:

Reply:
Yeah, it's not great that the software I actually want to put on bioconda ([DeepARG-SS](https://bitbucket.org/gusphdproj/deeparg-ss/src/master/)) depends on `nolearn`, which depends on `gdbn`, which depends on `gnumpy`, which is only py2 and hasn't been maintained since 2013